finance/finance-ctte-policy.rst

```diff
@@ -0,0 +1,58 @@
+##################################
+Interim Finance Committee Policies
+##################################
+
+This document records policies adopted by the `Astropy Interim Finance Committee`_ (hereafter “the Committee”).
```

Comment:
Maybe add a bit of context that this is a "running policy" document as opposed to something that's very fixed.